sims/vcs/Makefile

```diff
@@ -31,6 +31,22 @@ include $(base_dir)/vcs.mk
 default: $(sim)
 debug: $(sim_debug)
 
+#########################################################################################
+# simulaton requirements
+#########################################################################################
+SIM_FILE_REQS += \
+	$(ROCKETCHIP_RSRCS_DIR)/vsrc/TestDriver.v
+
+# ignore *.h files since vcs has +incdir
+$(sim_files): $(SIM_FILE_REQS)
+	mkdir -p $(dir $(sim_files))
```

Comment:
What is setting +incdir? Is it cleaner to just copy the header files as well?

Reply:
https://github.com/ucb-bar/chipyard/blob/45684f1b875f018a1945476cec9269068e1618ca/vcs.mk#L44

I should clarify this comment. I copy all files to the gen-src area but I don't include the headers in the `sim_files.h`.